src/astro/sql/operators/cleanup.py

```diff
@@ -213,3 +213,19 @@ def resolve_tables_from_tasks(
                     )
 
         return res
+
+
+def cleanup(tables_to_cleanup: Optional[List[Table]] = None, **kwargs):
+    """
+    Clean up temporary tables once either the DAG or upstream tasks are done
+
+    The cleanup operator allows for two possible scenarios: Either a user wants to clean up a specific set of tables
+    during the DAG run, or the user wants to ensure that all temporary tables are deleted once the DAG run is finished.
+    The idea here is to ensure that even if a user doesn't have access to a "temp" schema, that astro does not leave
+    hanging tables once execution is done.
+
+    :param tables_to_cleanup: A list of tables to cleanup, defaults to waiting for all upstream tasks to finish
+    :param kwargs:
+    :return:
+    """
+    return CleanupOperator(tables_to_cleanup=tables_to_cleanup, **kwargs)
```

Comment:
Does it make sense to also return the `.output` here too?

Reply:
If not, what does returning the operator bring us?

Reply:
🤷 - Probably just syntactic sugar to have "functions". We can return table objects that represent the deleted table but I wouldn't like to change the implementation in this PR as this PR just moves objects around